WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -140,6 +140,10 @@ class CommentDetailViewController: UIViewController, NoResultsViewHost {
         return cell
     }()
 
+    private weak var changeStatusViewController: UIViewController?
```

Comment:
If that's the presenting view controller, you can use `unowned` here and maybe specify the VCType rather than using `UIViewController` since the name already specifies it.

Reply:
That's the bottom sheet view controller being presented. I need to keep a reference to this view controller so I can dismiss it later when the user selects a status. The reference is optional because the view controller might not always be present.

While now it's possible to have optional unowned references:

```swift
private unowned var changeStatusViewController: BottomSheetViewController?
```

Unlike with `weak`, the reference won't automatically turn `nil` if the object is deallocated. Accessing a deallocated object will cause the app to crash.

I need the object to automatically turn `nil` when the view controller is deallocated ( aka dismissed ).

Reply:
> maybe specify the VCType rather than using UIViewController since the name already specifies it.

Resolved in 346b52c94bfc39b39125c576fb9e436b043ed52b